javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,129 @@
+let names = [];
+let todoList = [];
+//Main function of Voice Assistant
+function getReply(command) { 
+    if(command.startsWith('Hello')){
+        getResponse(command);
+    }else if(command.startsWith('What is my')){
+       getName(command);
+    }else if(command.startsWith('Add fishing')){
+        addToDoList(command);
+    }else if(command.startsWith('Add singing')){
+        updateToDoList(command);
```

Comment:
These had to be dynamic, so it could be anything and not just "fishing" or "singing".

Reply:
Yes, I have changed that to starts with and ends with